guardian/pkg/tunnel/tunnel.go

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) 2025 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tunnel
+
+import (
+	"context"
+	"io"
+	"net"
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calico/guardian/pkg/chanutil"
+)
+
+const (
+	tunnelNetwork = "voltron-tunnel"
```

Comment:
I wonder if there is any benefits to configure this at start up instead of hard-coding it. In case we want to change it, it would be better to not hard-coded.

Reply:
For now I'm just going to keep it hardcoded (since this is how it's currently done).